src/cljs/athens/page.cljs

```diff
@@ -100,11 +100,27 @@
          title]))))
 
 
+(defn merge-prompt
+  [{:keys [old-title new-title]}]
+  [:div {:style {:background "red"
+                 :color "white"}}
```

Comment:
Eventually we want all these styles in https://github.com/athensresearch/athens/blob/master/src/cljs/athens/style.cljs